### PR TITLE
DFTB+ path fix

### PR DESCRIPTION
**Pull request by t-young31**
- Fixes the default path used to execute DFTB+ (bug)
- Improves warning when loading a trajectory without a box/charge/spin multiplicity
- Adds an example using DFTB MD and comparison of force components to their true values over a quick trajectory

### CODE REVIEW

File: examples/additional_examples/mixed_energy_force/methane.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    return None
+
+
+def generate_energies_forces(trajectory):
+    """Calculate energies and forces for the frames at different levels of theory"""
+
+    frames = trajectory.copy()
+    frames.parallel_dftb()
+    frames.save(filename='dftb_frames.xyz')
+    np.savetxt('dftb_frames_F.txt', frames.force_components())
```
Comment (tristan-j-wood) [inline examples/additional_examples/mixed_energy_force/methane.py:28]: You have _F in this savetxt but not in the sequential files

File: examples/additional_examples/mixed_energy_force/methane.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    frames = trajectory.copy()
+    frames.parallel_orca()
+    frames.save(filename='mp2_frames.xyz')
+    np.savetxt('mp2_frames.txt', frames.force_components())
+
+    gt.GTConfig.orca_keywords = GradientKeywords(['CCSD(T)', 'def2-TZVP', 'EnGrad', 'NumGrad'])
+    frames = trajectory.copy()
+    frames.parallel_orca()
+    frames.save(filename='ccsdt_frames.xyz')
+    np.savetxt('ccsdt_frames_F.txt', frames.force_components())
```
Comment (tristan-j-wood) [inline examples/additional_examples/mixed_energy_force/methane.py:57]: Again you have _F but not in mps, pbe0, pbe, xtb

**Review by tristan-j-wood — COMMENTED** [review]
Just comments on filenames

File: examples/additional_examples/mixed_energy_force/methane.py (hunk 1 above)
Comment (t-young31) [reply examples/additional_examples/mixed_energy_force/methane.py:28]: whoops - thanks :+1:

**Pull request merged**